### PR TITLE
Filipino Translation - 1 126 translated words

### PR DESCRIPTION
**Pull request by mind-blast**
Translation the file reset.asc with total 1,126 translated words.
Note : code text and computer code text are not counted as translated words.

### CODE REVIEW

File: book/07-git-tools/sections/reset.asc [hunk 1]
```diff
@@ -1,35 +1,35 @@
 [[_git_reset]]
-=== Reset Demystified
+=== Ang Reset Demystified
 
-Before moving on to more specialized tools, let's talk about the Git `reset` and `checkout` commands.
-These commands are two of the most confusing parts of Git when you first encounter them.
-They do so many things that it seems hopeless to actually understand them and employ them properly.
-For this, we recommend a simple metaphor.
+Bago lumipat sa mas natatanging mga kasangkapan, makig-usap tayo tungkol sa Git ang `reset` at `checkout` na mga utos.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/reset.asc]: `makig-usap tayo tungkol` to `pag-usapan natin ang tungkol`

File: book/07-git-tools/sections/reset.asc [hunk 2]
```diff
@@ -1,35 +1,35 @@
...
+Ang mga utos na ito ay dalawang pinakanakakalito na mga parte sa Git na kapag ikaw ay magkasalubong sila.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/reset.asc]: kapag una mong nakasalubong sila

File: book/07-git-tools/sections/reset.asc [hunk 3]
```diff
@@ -1,35 +1,35 @@
...
 
-===== The HEAD
+===== Ang HEAD
 
-HEAD is the pointer to the current branch reference, which is in turn a pointer to the last commit made on that branch.
-That means HEAD will be the parent of the next commit that is created.
-It's generally simplest to think of HEAD as the snapshot of *your last commit on that branch*.
+Ang HEAD ay ang puntero sa kasalukuyang branch na reperensiya, na kung saan ay inikot ang puntero sa huling commit na nagawa sa branch na iyon.
+Ibig sabihin na ang HEAD ay maging magulang sa susunod na commit na nilikha.
+Sa pangkalahatan ang pinakasimple na maisip sa HEAD bilang ang snapshot sa *iyong huling commit sa branch na iyon*.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/reset.asc]: bilang snapshot - remove `ang`

File: book/07-git-tools/sections/reset.asc [hunk 4]
```diff
@@ -46,16 +46,16 @@ $ git ls-tree -r HEAD
 040000 tree 99f1a6d12cb4b6f19...   lib
 ----
 
-The Git `cat-file` and `ls-tree` commands are ``plumbing'' commands that are used for lower level things and not really used in day-to-day work, but they help us see what's going on here.
+Ang Git `cat-file` at `ls-tree` na mga utos ay ``plumbing'' na mga utos na ginagamit para sa ibabang antas ng mga bagay at hindi talagang ginagamit sa araw-araw na trabaho, ngunit sila ay nakakatulong sa atin kung ano ang nangyari dito.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/reset.asc]: - mas mababang antas
- nakakatulong sa atin upang makita kung ano ang nangyayari dito.

File: book/07-git-tools/sections/reset.asc [hunk 5]
```diff
@@ -46,16 +46,16 @@ $ git ls-tree -r HEAD
...
+===== Ang Index
 
-The Index is your *proposed next commit*.
-We've also been referring to this concept as Git's ``Staging Area'' as this is what Git looks at when you run `git commit`.
+Ang Index ay ang iyong *iminungkahing susunod na commit*.
+Nag-uulat din kami sa konseptong ito bilang Git na ``Staging na Lawak'' dahil ito ay kung ano ang tingin ng Git ay makikita kapag ikaw ay nagpatakbo ng `git commit`.
 
-Git populates this index with a list of all the file contents that were last checked out into your working directory and what they looked like when they were originally checked out.
-You then replace some of those files with new versions of them, and `git commit` converts that into the tree for a new commit.
+Ang Git ay nagpaparaming nitong index na may isang listahan sa lahat ng mga nilalaman ng file na huling na tingnan sa iyong tinatrabahong direktoryo at kung ano ang hitsura nila kapag sila ay orihinal na naka-checkout.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/reset.asc]: Ang Git ay nagpaparami ng index na ito

File: book/07-git-tools/sections/reset.asc [hunk 6]
```diff
@@ -65,16 +65,16 @@ $ git ls-files -s
 100644 47c6340d6459e05787f644c2447d2595f5d3a54b 0	lib/simplegit.rb
 ----
 
-Again, here we're using `git ls-files`, which is more of a behind the scenes command that shows you what your index currently looks like.
+Muli, dito kami ay gumagamit ng `git ls-files`, na kung saan ay  higit pa sa isang likod ng mga eksekna na utos na nagpapakita sa iyo kung ano ang mukha ng iyong index.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/reset.asc]: - dito kami gumagamit
- only one space between `ay` and `higit`

File: book/07-git-tools/sections/reset.asc [hunk 7]
```diff
@@ -88,79 +88,79 @@ $ tree
...
 
-Git's main purpose is to record snapshots of your project in successively better states, by manipulating these three trees.
+Ang pangunahing layunin sa Git ay ang pagtatala ng mga snapshot sa iyong proyekto sa sunod-sunod na mas mabuting estado, sa pamamagitan ng pagmamanipula ng mga tatlong mga tree.
 
 image::images/reset-workflow.png[]
 
-Let's visualize this process: say you go into a new directory with a single file in it.
-We'll call this *v1* of the file, and we'll indicate it in blue.
-Now we run `git init`, which will create a Git repository with a HEAD reference which points to an unborn branch (`master` doesn't exist yet).
+I-visualize natin ang prosesong ito: sabihin mong pumasok ka sa isang bagong direktoryo na may isang solong file sa loob.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/reset.asc]: translate `I-visualize` to `Isalarawan`

File: book/07-git-tools/sections/reset.asc [hunk 8]
```diff
@@ -88,79 +88,79 @@ $ tree
...
-At this point, only the Working Directory tree has any content.
+Sa puntong ito, Tanging ang Tinatrabahong Direktoryo na tree ay merong anumang nilalaman.
 
-Now we want to commit this file, so we use `git add` to take content in the Working Directory and copy it to the Index.
+Ngayon gusto naming i-commit ang file na ito, kaya gumamit kami ng `git add` upang kunin ang nilalaman ng Tinatrabahong Direktoryo at kopyahin ito sa Index.
 
 image::images/reset-ex2.png[]
 
-Then we run `git commit`, which takes the contents of the Index and saves it as a permanent snapshot, creates a commit object which points to that snapshot, and updates `master` to point to that commit.
+Pagkatapos ay pinatakbo namin ang `git commit`, na tumatagal ng mga nilalaman sa Index at sine-save ito bilang isang permanenteng snapshot, lumilikha ng isang commit na bagay na kung saan ay nagtuturo sa snapshot na iyon, at i-update ang `master` upang ituro ang commit na iyon.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/reset.asc]: change `tumatagal` to `kumukuha`

File: book/07-git-tools/sections/reset.asc [hunk 9]
```diff
@@ -88,79 +88,79 @@ $ tree
...
 
-If we run `git status`, we'll see no changes, because all three trees are the same.
+Kung magpatakbo ng `git status`, hindi kami makakita ng mga pagbabago, dahil lahat ng tatlong mga tree ay pareho lang.
 
-Now we want to make a change to that file and commit it.
-We'll go through the same process; first we change the file in our working directory.
-Let's call this *v2* of the file, and indicate it in red.
+Ngayon gusto naming gumawa ng isang pagbabago sa file na iyon at i-commit ito.
+Kami ay dumadaan sa parehong proseso; una ay binago natin ang file sa aming tinatrabahong direktoryo.
+Tawagin na itong *v2* sa file, at ipahiwatig ito sa pula.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/reset.asc]: Tawagin natin itong

File: book/07-git-tools/sections/reset.asc [hunk 10]
```diff
@@ -88,79 +88,79 @@ $ tree
...
-If we run `git status` right now, we'll see the file in red as ``Changes not staged for commit,'' because that entry differs between the Index and the Working Directory.
-Next we run `git add` on it to stage it into our Index.
+Kung patakbuhin natin ang `git status` ngayon na, makikita natin ang file sa pula bilang ``Pagbabago ay hindi naka-stage para commit,'' dahil ang entry na ito ay naiiba ang Index at ang Tinatrabahong Direktoryo.
+Susunod pinatakbo namin ang `git add` dito upang i-stage ito sa ating Index.
 
 image::images/reset-ex5.png[]
 
-At this point, if we run `git status`, we will see the file in green under ``Changes to be committed'' because the Index and HEAD differ -- that is, our proposed next commit is now different from our last commit.
-Finally, we run `git commit` to finalize the commit.
+Sa puntong ito, kung ipatakbo namin ang `git status`, makikita natin ang file sa berde sa ilalim ng ``Pagbabago na i-commit'' dahil ang Index at HEAD na pagkaiba -- yan ay, ang aming iminungkahi sa susunod na commit ay ngayon naiiba mula sa ating huling commit.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/reset.asc]: sa susunod na commit na ngayon ay naiiba

File: book/07-git-tools/sections/reset.asc [hunk 11]
```diff
@@ -88,79 +88,79 @@ $ tree
...
 
 image::images/reset-ex6.png[]
 
-Now `git status` will give us no output, because all three trees are the same again.
+Ngayon ang `git status` ay magbibigay sa amin nang walang output, dahil lahat ng tatlong mga tree ay pareho muli.
 
-Switching branches or cloning goes through a similar process.
-When you checkout a branch, it changes *HEAD* to point to the new branch ref, populates your *Index* with the snapshot of that commit, then copies the contents of the *Index* into your *Working Directory*.
+Paglipat ng mga branch o pag-clone ay dumadaan sa isang katulad na proseso.
+Kapag ikaw ay nag-checkout sa isang branch, ito ay nagbabago ng *HEAD* upang ituro sa bagong branch na ref, nagpapalaganap na iyong *Index* na gamit ang snapshot sa commit na iyon, pagtapos ay kumukopya sa mga nilalaman sa *Index* sa iyong *Tinatrabahong Direktoryo*.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/reset.asc]: pagkatapos ay kumukopya

File: book/07-git-tools/sections/reset.asc [hunk 12]
```diff
@@ -88,79 +88,79 @@ $ tree
...
 
-==== The Role of Reset
+==== Ang Tungkilin ng Reset
 
-The `reset` command makes more sense when viewed in this context.
+Ang `reset` na utos ay mas makakatulong kapag tiningnan sa konetekstong ito.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/reset.asc]: kontekstong

**Review by josteinfrancois — COMMENTED** [review]
Hey @robin-ho, good day! I have reviewed your work and made a few minor suggestions. 
I also have another concern regarding the usage of "namin" and "amin". You could use "natin" and "atin" in there place. This is to fully engage the narrator-reader relationship in the text. Anyway, that is only a suggestion. Thank you.

File: book/07-git-tools/sections/reset.asc (hunk 1 above)
Comment (mind-blast) [reply book/07-git-tools/sections/reset.asc]: done change

File: book/07-git-tools/sections/reset.asc (hunk 2 above)
Comment (mind-blast) [reply book/07-git-tools/sections/reset.asc]: done change

File: book/07-git-tools/sections/reset.asc (hunk 3 above)
Comment (mind-blast) [reply book/07-git-tools/sections/reset.asc]: done removing

File: book/07-git-tools/sections/reset.asc (hunk 4 above)
Comment (mind-blast) [reply book/07-git-tools/sections/reset.asc]: done change

File: book/07-git-tools/sections/reset.asc (hunk 5 above)
Comment (mind-blast) [reply book/07-git-tools/sections/reset.asc]: done change

File: book/07-git-tools/sections/reset.asc (hunk 6 above)
Comment (mind-blast) [reply book/07-git-tools/sections/reset.asc]: done change

File: book/07-git-tools/sections/reset.asc (hunk 7 above)
Comment (mind-blast) [reply book/07-git-tools/sections/reset.asc]: done change

File: book/07-git-tools/sections/reset.asc (hunk 8 above)
Comment (mind-blast) [reply book/07-git-tools/sections/reset.asc]: done change

File: book/07-git-tools/sections/reset.asc (hunk 9 above)
Comment (mind-blast) [reply book/07-git-tools/sections/reset.asc]: done change

File: book/07-git-tools/sections/reset.asc (hunk 10 above)
Comment (mind-blast) [reply book/07-git-tools/sections/reset.asc]: done change

File: book/07-git-tools/sections/reset.asc (hunk 11 above)
Comment (mind-blast) [reply book/07-git-tools/sections/reset.asc]: done change

File: book/07-git-tools/sections/reset.asc (hunk 12 above)
Comment (mind-blast) [reply book/07-git-tools/sections/reset.asc]: done change

**Comment by mind-blast** [comment]
I have revised as suggested, I have changed also to natin . Thank you for the proof read.

**Pull request merged**